bitnami/redis/templates/sentinel/pdb.yaml

```diff
@@ -3,7 +3,7 @@ Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if and (eq .Values.architecture "replication") (not .Values.sentinel.enabled) .Values.replica.pdb.create }}
+{{- if and (eq .Values.architecture "replication") .Values.sentinel.enabled .Values.replica.pdb.create }}
```

Comment:
For backwards compatibility, this needs to be:

```
{{- $pdb := coalesce .Values.replica.pdb .Values.pdb }}
{{- if and (eq .Values.architecture "replication") .Values.sentinel.enabled $pdb.create }}
```

Chart version 19.3.0 used to create a PDB for sentinel nodes based on `.Values.pdb`.

Reply:
Version 19.3.0 was using the same PDB for all pods created by this helm chart, without any distinction between replicas, master or sentinel pods, what was a failure. Anyway I've just changed the templates to add backwards compatibility